app/containers/message/Components/Attachments/Thumbnail.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { useEffect, useState } from 'react';
+import { StyleSheet, View } from 'react-native';
+
+import { getThumbnailAsync } from 'expo-video-thumbnails';
+import FastImage from 'react-native-fast-image';
+
+import { CustomIcon } from '../../../CustomIcon';
+import OverlayComponent from '../OverlayComponent';
+
+const styles = StyleSheet.create({
+	container: {
+		flex: 1,
+		alignItems: 'center',
+		justifyContent: 'center'
+	},
+	overlay: {
+		flex: 1
+	},
+	image: {
+		width: '100%',
+		height: '100%'
+	},
+	playerIcon: {
+		position: 'absolute',
+		shadowColor: '#000',
+		shadowOpacity: 0.3,
+		shadowOffset: {
+			width: 1,
+			height: 1
+		}
+	}
+});
+
+type Image = {
+	loading: boolean;
+	uri: string | null;
+};
+
+type ThumbnailProps = {
+	url: string;
+	encrypted?: boolean;
+};
+
+const Thumbnail = ({ url, encrypted = false }: ThumbnailProps) => {
```

Comment:
You asked my opinion about splitting Thumbnail in a separate file.
I said we should either keep on Video file (it's a small file, so it's ok) or we create a Video folder so `message/Components/Attachments` is organized with one file per feature.

You create `Thumbnail` file without a Video folder, so Attachments now have 
```
- CollapsibleQuote (folder)
- Image (folder) 
- AttachedActions.tsx
- Attachments.tsx
- Audio.tsx
- Reply.tsx
- Video.tsx
- Thumbnail.tsx (this one is only related to Video)
```